# exceptions.py


class ioka_api_error(Exception):
    pass
